### PR TITLE
cleanup of BrainAtlas instances

### PR DESCRIPTION
**Pull request by lzehl**
related to #240 

This is a basic cleanup including updates of:
+ add new properties (usedSpecies)
+ change property names (hasTerminology/definedIn --> hasTerminology/dataLocation)
+ change value types (homepage: obj (URL) --> homepage: string (IRI))
+ fix indentation (should be 2 spaces)
+ fixing names/ids for consistency
 
Checklist for current brain atlas instances:
- [x] AAL1
- [x] AllenMouseBrainAtlas
- [x] AsegAtlas
- [x] BrodmannAreasHuman
- [x] DeepWhiteMatterAtlas
- [x] Desikan-KillianyAtlas
- [x] Julich-BrainAtlas
      + NOTE: add missing areas of new release (v3.0/v3.0.1)
- [x] Schaefer400p7n
      + NOTE: delete BA only keep BAV for now
- [x] WaxHolmSpacesSDRatBrainAtlas
      + NOTE: missing areas of new release are not yet added

ATTENTION for ID changes (will require cleanup in EBRAINS):
+ AsegAtlas (BA ID + Entity IDs)
+ BrodmannAreasHuman (BA ID + Entity IDs)
+ DeepWhiteMatterAtlas (BA ID)
+ SchaeferAtlas (BA deleted; needs to be replaced with respective BAV ID + PEV IDs)

### CODE REVIEW

**Comment by lzehl** [comment]
@aeidi89 I think the solution for the Schaefer Atlas (and the DiFuMo) does not work or at least would have the following consequence:

Parcels are name e.g. Vis_1, Vis_2, Vis_3, Vis_4, ... SomMot_1, SomMot_2, SomMot_3,... across the different parcellation versions 400p, 600p7N, 800p, 1000p. But Vis_1 of 400p is not the same as Vis_1 in 1000p. Even between network versions I can't track the parcellation entity names because they are renamed to VisCent_ExStr_1, VisCent_ExStr_2, VisCent_ExStr_3, VisPeri_StriCal_1, VisPeri_ExStrInf_1, VisPeri_ExStrSup_1, etc. 

That means that either I basically do not have a version independent semantic term for a parcellation entity, correct?

File: exit
Comment (aeidi89) [inline exit:1]: This file I don't understand @lzehl 
What does this mean?

**Review by aeidi89 — COMMENTED** [review]
Made a few minor changes in direct submit. 
Looks good, but I don't understand the first file called "exit"

**Comment by lzehl** [comment]
@aeidi89 not sure about the file. I think this was a hidden file I've created and pushed by accident. I deleted it. Feel free to merge when you are otherwise happy.

**Pull request merged**